Package.swift

```diff
@@ -31,11 +31,11 @@ let package = Package(
   dependencies: [
     .package(url: "https://github.com/apple/swift-collections", from: "1.0.0"),
     .package(url: "https://github.com/swiftlang/swift-docc-plugin", from: "1.0.0"),
-    .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "1.5.4"),
+    .package(url: "https://github.com/pointfreeco/swift-case-paths", branch: "main"),
```

Comment:
@gemmakbarlow I just cut a release for Case Paths. Can you bump this and update the Package.resolved file?

```suggestion
    .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "1.5.6"),
```

Reply:
Done !